src/WebUI/Components/ServicesComponent.scss

```diff
@@ -0,0 +1,32 @@
+@import "azure-devops-ui/Core/_platformCommon.scss";
+
+.sc-list-content{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    justify-content: space-around;
+}
+
+.sc-col-header {
+    color: $secondary-text;
+    font-size: $fontSizeM;
+}
+
+.sc-col-content {
+    display: inline-flex;
+    align-items: center;
+    font-size: 16px;
+}
+
+.sc-wide-col {
+    width: 140px;
+}
+
+.sc-col-primary-text {
+    font-weight: $fontWeightSemiBold;
+    font-size: $fontSizeL;
+}
+.sc-col-secondary-text {
+    color: $secondary-text;
+    font-size: $fontSizeM;
+}
```

Comment:
sc-col-primary-text and secondary can be used in other places, in future we may move this to common place. anyway for the time being keep it here.